pkg/resources/monitoring/dashboards/home.json

```diff
@@ -22,7 +22,7 @@
       {
         "datasource": {
           "type": "prometheus",
-          "uid": "P0BEE70D62F42845E"
+          "uid": "${DS_PROMETHEUS}"
```

Comment:
Where is `${DS_PROMETHEUS}` referenced?

Reply:
Here: https://github.com/rancher/opni/blob/9b4df775c892522baa8a68489b1faa226f8343f6/pkg/resources/monitoring/grafana.go#L478-L481

It creates a UID/input name used by the dashboards to specify from which data source some data comes.